dcpy/lifecycle/ingest/transform.py

```diff
@@ -109,3 +111,158 @@ def to_parquet(config: Config, local_data_path: Path | None = None):
 
     gdf.to_parquet(PARQUET_PATH, index=False)
     logger.info(f"✅ Converted raw data to parquet file and saved as {PARQUET_PATH}")
+
+
+class Preprocessors:
```

Comment:
These functions are very half-baked. I'd like to just leave them for now, and iterate on them while trying to actually get this working for our template datasets